oxidation/bindings/README.md

```diff
@@ -19,24 +33,77 @@ npm run build  # Generate index.node (basically a .so that node can load)
 
 ## 2. Android
 
-Requirements:
+### Requirements
+
+You can either install `Android Studio` or `Android tools`
+
+#### Permission issue
+
+If you have permission issue when installing the packages `android sdk` or `android ndk`.
+It indicate that you can't write to the `ANDROID_HOME` (or `ANDROID_SDK_HOME` (they should be the same value)).
+
+1. Copy `ANDROID_HOME` to a place in your computer where you have `write` access.
+2. Update `ANDROID_HOME` & `ANDROID_SDK_HOME` env variables to the new path.
+
+#### Install Android Studio
+
+1. Download Android studio at <https://developer.android.com/studio#android-studio-downloads>
+2. Install Android Studio
+3. In `File > Settings > Appearance & Behavior > System Settings > Android SDK > SDK Platform > Show Package Details` :
+
+    - Install Android SDK `30.0.3`
+    - Install Android NDK `22.1.7171670`
+    - Android SDK Command-line Tools
+
+4. Python command should be available in PATH environment variable (used by `rust-android-gradle` plugin)
+
+#### Install Android tools
+
+1. Download android tools at <https://developer.android.com/studio#command-line-tools-only>
+2. Ensure you have `openjdk-8` installed (it is required to run `android-tools`'s commands).
+3. Install Android `SDK-30.0.3` and `NDK-22.1.7171670` by running
+
+    ```shell
+    JAVA_HOME=/jdk8/path sdkmanager --install "ndk;<ndk version>" "build-tools;<build tool version>"
+    ```
 
-- Install Android Studio
-- In `File > Settings > Appearance & Behavior > System Settings > Android SDK > SDK Platform > Show Package Details` : Install Android SDK 30.0.3
-- In `File > Settings > Appearance & Behavior > System Settings > Android SDK > SDK Tools > Show Package Details` : Install Android NDK 22.1.7171670 and Android SDK Command-line Tools
-- Python command should be available in PATH environment variable (used by `rust-android-gradle` plugin)
+#### Install required rust targets
 
 Install Rust targets for cross compilation:
 
-```bash
-rustup target add armv7-linux-androideabi
-rustup target add i686-linux-android
-rustup target add aarch64-linux-android
-rustup target add x86_64-linux-android
+```shell
+rustup target add \
+  armv7-linux-androideabi \
+  i686-linux-android \
+  aarch64-linux-android \
+  x86_64-linux-android \
+```
+
+> The `bindings/android` directory is a valid Android project to generate a libparsec AAR.
+> However the AAR doesn't have to be built when using `../client/android` project (given it depends explicitly on the `bindings/android/libparsec`).
+
+#### Install Java development kit to build the application
+
+Having installed `openjdk-8` in the step [Install Android tools](#install-android-tools) wont be enough to build the android apps.
```

Comment:
So we need two different version of openjdk ?

I can run just fine android tools with the default jre on ubuntu (i.e. openjdk 11)

so I would say we just need a recent enough version of openjdk, as we currently have no configuration in Gradle to specify a specific version

Reply:
Strange, mine don't like it :thinking: I will test more that one